docs/requirements.md

```diff
@@ -5,8 +5,7 @@ storage and must have network connectivity with the GitHub Enterprise Server app
 
 ## Backup host requirements
 
-Backup host software requirements are modest: Linux or other modern Unix operating
-system (Ubuntu is highly recommended) with [bash][1], [git][2], [OpenSSH][3] 5.6 or newer, [rsync][4] v2.6.4 or newer, and [jq][11] v1.5 or newer.
+Backup host software requirements are modest: Ubuntu Linux operating system with [bash][1], [git][2], [OpenSSH][3] 5.6 or newer, [rsync][4] v2.6.4 or newer, and [jq][11] v1.5 or newer.
```

Comment:
Nice!

Reply:
Does this mean using Redhat/Alma as a backup server will no longer be supported?

Reply:
@eisenhowerj : Thanks for the question. I'm actually in the process of updating the requirements section to make the result of our internal discussion clear. I updated the PR to `draft` while I make that change.

Reply:
@eisenhowerj : I have updated this PR to make it clearer how non-Ubuntu operating systems are supported. As a result I'll mark this conversation as resolved.